### ISSUE TITLE
Accessibility: Focus does not return to triggering element when modal is closed

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51925]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

When a modal is closed, keyboard focus should be returned to the button that triggered it so keyboard-only and screen reader users do not lose their place on the page.

When this modal closes, the keyboard focus is not returned to the triggering button. Instead, it is shifted to the footer of the web page. Hitting the Tab key once places users into the browser's address bar, and they must tab through many elements to return to where they were on the page.

![Image](https://github.com/user-attachments/assets/486504c1-94f3-418a-80e6-f9b8890422f7)

### Component or Page: 

Page: Product Single (Simple)  

### Name affected element:

Zoom product image link

### Relevant code snippets:

`<a href="#" class="woocommerce-product-gallery__trigger">🔍</a>`

### Recommended fix:

Use JavaScript to return keyboard focus to the button that opened the modal when it is closed.

**Comment by darrenjacoby** [comment]
@vikrampm1 @dkotter this appears to be working now on my local, do we know if it was resolved in another task?

**Comment by dkotter** [comment]
> [@vikrampm1](https://github.com/vikrampm1) [@dkotter](https://github.com/dkotter) this appears to be working now on my local, do we know if it was resolved in another task?

I'm unsure if this was fixed somewhere else but if you're not seeing it anymore, I think we pass this over to our QA team to verify and if they can't reproduce, we can close this out.

@ankitguptaindia @qasumitbagthariya See above for details but this issue may no longer exist so need to verify that

**Comment by Manussakis** [comment]
@dkotter @darrenjacoby @vikrampm1 This issue was fixed with the https://github.com/woocommerce/woocommerce/pull/50730 PR.
I just linked them.

**Comment by vikrampm1** [comment]
Yup, looks like the exact issue so I am proceeding with closing the issue.
